test/test_p4est_2d.jl

```diff
@@ -198,14 +198,16 @@ isdir(outdir) && rm(outdir, recursive=true)
       tspan = (0.0, 0.0001),
       rtol = 1.0e-7,
       skip_coverage=true)
-    # Ensure that we do not have excessive memory allocations 
-    # (e.g., from type instabilities) 
-    let 
-      t = sol.t[end] 
-      u_ode = sol.u[end] 
-      du_ode = similar(u_ode) 
-      @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000 
-    end
+      #=
+      # Ensure that we do not have excessive memory allocations 
+      # (e.g., from type instabilities) 
+      let 
+        t = sol.t[end] 
+        u_ode = sol.u[end] 
+        du_ode = similar(u_ode) 
+        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000 
+      end
+      =#
```

Comment:
Why did you remove these tests again? Keeping code commented out is not an option. We should choose one of the following options

- keep the code in a working state if the tests pass
- mark the tests as broken if they don't work
- (remove the tests)

Reply:
The reason is the `skip_coverage=true` option which causes the test to be skipped during coverage run, which causes the test then to fail in a subsequent run.

Reply:
I added `if @isdefined sol` to guard the allocation check, this works now hopefully.

Reply:
Thanks for the explanation! Could you please add a comment to the relevant places explaining this?

Reply:
Sure :+1: